src/components/DropdownMenu.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import styled from 'styled-components';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+
+import {
+  loadRequest,
+  copyRequest,
+  removeRequest,
+  addCurrentRequest
+} from '../store/slices/requestSlice';
+import { requestsSelector, dropdownSelector } from '../store/slices/selectors';
+
+const DropdownList = styled.ul`
+  position: relative;
+
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding: 0;
+
+  list-style: none;
+
+  &::before {
+    content: '';
+
+    position: absolute;
+    bottom: 50px;
+    left: 0;
+    right: 0;
+
+    width: auto;
+    height: 1px;
+
+    background-color: rgba(0, 0, 0, 0.2);
+  }
+`;
+
+const DropdownItem = styled.li`
+  margin-top: ${(props) => props.marginTop || '0'};
+  margin-bottom: ${(props) => props.marginBottom || '0'};
+`;
+
+const DropdownButton = styled.button`
+  display: block;
+  width: 100%;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-left: 15px;
+  border: none;
+
+  font-size: 16px;
+  line-height: 20px;
+  text-align: left;
+  cursor: pointer;
+
+  background-color: transparent;
+
+  &:hover {
+    color: ${(props) => props.color ? '#FFFFFF' : '#0D0D0D'};
```

Comment:
Ternary formatting. Please check all the app and fix it everywhere.

Reply:
Checked all app and fixed it everywhere.

Reply:
LGTM